### PR TITLE
feat: ei-1288 | Add GetHostID rpc to new HostAPI service

### PR DESCRIPTION
**Pull request by bernardcooke-iotics**

### CODE REVIEW

File: proto/iotics/api/host.proto [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+option php_namespace = "Iotics\\Api";
+
+// ---------------------------------------------------------------------------------------------------------------------
+
+
+// HostAPI enables management of Iotics host twins.
+// Services only affect local resources, unless stated otherwise.
+service HostAPI {
+  // GetHostTwinID gets the ID of the host twin.
+  rpc GetHostTwinID(GetHostTwinIDRequest) returns (GetHostTwinIDResponse) {}
```
Comment (mihai-dinculescu) [inline proto/iotics/api/host.proto]: It would be good to maintain the consistency and call this `HostTwinDid`.

Comment (bernardcooke-iotics) [reply proto/iotics/api/host.proto]: @mihai-dinculescu at the moment it will return PeerID though, until @conbon makes some other changes

Comment (bernardcooke-iotics) [reply proto/iotics/api/host.proto]: So what I mean to say is `HostTwinID` vs `HostTwinDID`?

Comment (conbon) [reply proto/iotics/api/host.proto]: I thought we were going with `hostId` so the switchover is transparent and it matches with the coming api changes

Comment (mihai-dinculescu) [reply proto/iotics/api/host.proto]: If it's still peer id, then yes, `HostTwinID` makes sense.

File: proto/iotics/api/host.proto [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+// GetHostTwinIDRequest: gets the local host twin's ID
+message GetHostIDRequest {
+  Headers headers = 1;
+}
+
+// GetHostTwinIDResponse: response containing the local host twin's ID
+message GetHostIDResponse {
+  message Payload {
+    HostID hostId = 1;
```
Comment (conbon) [inline proto/iotics/api/host.proto]: This can be a string - HostID wrapper will disappear

File: proto/iotics/api/host.proto [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  // GetHostTwinID gets the ID of the host twin.
+  rpc GetHostID(GetHostIDRequest) returns (GetHostIDResponse) {}
+}
+
+// GetHostTwinIDRequest: gets the local host twin's ID
+message GetHostIDRequest {
+  Headers headers = 1;
+}
+
+// GetHostTwinIDResponse: response containing the local host twin's ID
```
Comment (conbon) [inline proto/iotics/api/host.proto]: Updates comments

File: proto/iotics/api/host.proto [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+option objc_class_prefix = "IAX";
+option php_namespace = "Iotics\\Api";
+
+// ---------------------------------------------------------------------------------------------------------------------
+
+
+// HostAPI enables management of Iotics host twins.
+// Services only affect local resources, unless stated otherwise.
+service HostAPI {
+  // GetHostTwinID gets the ID of the host twin.
```
Comment (StephanieBracaloni) [inline proto/iotics/api/host.proto]: Comment `GetHostTwinID` => `GetHostID gets the ID of the host.`

**Pull request merged**